src/backend_model.cc

```diff
@@ -207,6 +206,46 @@ TritonModel::Create(
     }
   }
 
+  // Initalize the custom batching library for the model, if provided.
+  if (model_config.has_sequence_batching()) {
+    if (model_config.parameters().contains("TRITON_BATCH_STRATEGY_PATH")) {
+      LOG_ERROR << "TRITON_BATCH_STRATEGY_PATH cannot be specified with "
```

Comment:
It might be more useful to return an error here rather than logging an error as this notifies the user of an invalid config. In the case where they want to keep tritonserver running even with failures, they can use `--exit-on-error=false` and they can try again.

Reply:
Good call, Kyle! Fixed to return an error.